### PR TITLE
Adds ogre ergo split

### PR DESCRIPTION
**Pull request by ctrlShiftBryan**
## Types of Changes

- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ogre/ergo_split/ergo_split.h [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  k63, k00, k10, k20, k30, k40, k50, k60, \
+  k65, k55, k45, k35, k25, k15, k05, k68, \
+  k01, k11, k21, k31, k41, k51, k61, \
+  k66, k56, k46, k36, k26, k16, k06, \
+  k02, k12, k22, k32, k42, k52, k62, \
+  k67, k57, k47, k37, k27, k17, k07, \
+  k03, k13, k23, k33, k43, k53,  \
+  k58, k48, k38, k28, k18, k08, \
+  k04, k14, k24, k34, k44, k54, k64, \
+  k69, k59, k49, k39, k29, k19, k09 \
```
Comment (noroadsleft) [inline keyboards/ogre/ergo_split/ergo_split.h]: Lines 22-31 should be arranged to resemble the layout used for your layers in `keymap.c`.

File: keyboards/ogre/ergo_split/rules.mk [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+# If you don't know the bootloader type, then you can specify the
+# Boot Section Size in *bytes* by uncommenting out the OPT_DEFS line
+# Otherwise, delete this section
+#   Teensy halfKay      512
+#   Teensy++ halfKay    1024
+#   Atmel DFU loader    4096
+#   LUFA bootloader     4096
+#   USBaspLoader        2048
+# OPT_DEFS += -DBOOTLOADER_SIZE=4096
+
```
Comment (noroadsleft) [inline keyboards/ogre/ergo_split/rules.mk]: Please delete Lines 14-23.

File: keyboards/ogre/ergo_split/rules.mk [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no
+EXTRAKEY_ENABLE = no
+CONSOLE_ENABLE = no
+COMMAND_ENABLE = no
```
Comment (noroadsleft) [inline keyboards/ogre/ergo_split/rules.mk]: Please include the rule comments on Lines 28-31:

```Makefile
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = no        # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
```

File: keyboards/ogre/ergo_split/keymaps/default/config.h [hunk 4]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2020 ctrlshiftba
```
Comment (fauxpark) [inline keyboards/ogre/ergo_split/keymaps/default/config.h]: This file can be deleted.

File: keyboards/ogre/ergo_split/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+            KC_ESC,  KC_A,    KC_S,    KC_D,    KC_F,   KC_G,   KC_PGDN, KC_QUOT, KC_H,   KC_J,   KC_K,    KC_L,   KC_SCLN, TP_QUOT, \
+            KC_LSFT, TP_Z,    TP_X,    KC_C,    KC_V,   KC_B,                     KC_N,   KC_M,   KC_COMM, KC_DOT, KC_SLSH, KC_RSFT, \
+            TP_GR,   KC_LALT, KC_LEFT, KC_RGHT, TP_SPC, KC_SPC, KC_BSPC, KC_RALT, KC_ENT, TP_ENT, KC_DOWN, KC_UP,  KC_LBRC, TP_RCTRL \
+  ),
+  [1] = LAYOUT(
+    RESET,  _______, KC_F1,   KC_F2,   KC_F3,   KC_F4,    KC_F5,  KC_INS,  KC_HOME, KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11, KC_F12, \
+            RGB_MOD, RGB_HUI, RGB_SAI, RGB_VAI, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+            RGB_M_P, RGB_HUD, RGB_SAD, RGB_VAD, _______, _______, KC_END,  _______, _______, _______, _______, _______, _______, _______, \
+            RGB_TOG, _______, _______, _______, _______, _______,                   _______, _______, _______, _______, _______, _______, \
+            _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,  KC_MENU, _______ \
```
Comment (fauxpark) [inline keyboards/ogre/ergo_split/keymaps/default/keymap.c]: The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**